developers/integrations/llm-frameworks/llamaindex/LlamaIndex.md

```diff
@@ -0,0 +1,44 @@
+---
+title: LlamaIndex
+sidebar_position: 5
+image: og/docs/llamaindex.jpg
+---
+
+[LlamaIndex](https://www.llamaindex.ai/) is a framework for building large language model (LLM) applications. 
+
+## LlamaIndex and Weaviate
+Weaviate is a [supported vector store](https://docs.llamaindex.ai/en/stable/api_reference/storage/vector_store/weaviate/) in LlamaIndex. You need a running Weaviate instance to build the vector store. You can create it with:
```

Comment:
Consider

Weaviate is a [supported vector store](https://docs.llamaindex.ai/en/stable/api_reference/storage/vector_store/weaviate/) in LlamaIndex.

Create a vector store for LlamaIndex: